storageFormats/eventMetadata.h

```diff
@@ -112,6 +115,31 @@ namespace rpwa {
 		return metadata.print(out);
 	}
 
+
+	/// \brief Handles additional saved variables of event data
```

Comment:
maybe one could elaborate a bit more:
class represents kinematic variables used for binning and cutting
provides interface to connect kinematic variables to tree branches and to check whether event is within a kinematic cell